server/container/guice/cassandra-guice/README.adoc

```diff
@@ -26,20 +32,25 @@ An usual compilation using maven will produce two artifacts into target director
 
 To run james, you have to create a directory containing required configuration files.
 
-A sample directory (appropriately named sample-directory) is provided with some
-default value you may need to replace.
+James requires the configuration to be in a subfolder of working directory that is called **conf**. You can get a sample
+directory for configuration from dockerfiles/run/guice/cassandra/destination/conf. You might need to adapt it to your needs.
 
 You also need to generate a keystore with the following command :
 [source]
 ----
-$ keytool -genkey -alias james -keyalg RSA -keystore conf/keystore
+$ keytool -genkey -alias james -keyalg RSA -keystore sample-configuration/keystore
 ----
 
-You need to have a Cassandra and an ElasticSearch instance running.
+You need to have a Cassandra and an ElasticSearch instance running. You can either launch them via docker or install the servers :
```

Comment:
you should invert the or clause, it let think that the next commands are installing the servers